functions/list.go

```diff
@@ -1,18 +1,9 @@
 package functions
 
 import (
-	"encoding/json"
-	"errors"
 	"fmt"
-	"io"
-	"net/http"
 
-	"github.com/devilcove/httpclient"
 	"github.com/gravitl/netclient/config"
-	"github.com/gravitl/netmaker/logger"
-	"github.com/gravitl/netmaker/models"
-	"github.com/kr/pretty"
-	"golang.zx2c4.com/wireguard/wgctrl/wgtypes"
 )
 
 type Peer struct {
```

Comment:
this and CIDR struct below are missing comments

Reply:
structs were redundant, removed